todoman/ui.py

```diff
@@ -251,8 +259,18 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        :param datetime.date date: a datetime object
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        * if no date is supplied, it returns empty_date
+        """
         if date:
-            rv = date.strftime(self.date_format)
+            if date in self.special_dates:
```

Comment:
Yes, sorry, this is something we've dragged for a while. #109 will fix this in future, but for now, just update this to `date.date()`.